kmod/src/server.c

```diff
@@ -4095,3 +4095,11 @@ void scoutfs_server_destroy(struct super_block *sb)
 		sbi->server_info = NULL;
 	}
 }
+
+void scoutfs_server_net_shutdown(struct super_block *sb)
```

Comment:
It's not safe to reference the server-> fields here like it.. kind of is in the client fields.   The client conn is a long lived container that is up for the duration of the mount.   If we're calling umount_begin then we've built up the mount and haven't torn it down yet -- we know the conn is still around.   The server is on a totally different life cycle.  It's built up and torn down as quorum is elected leader and the server decides to shutdown for whatever reason it wants.   forced unmont can race with the server shutting down and load a valid server_info pointer that is being actively freed while the server is shutting down.  Happily, I don't think we have to actively shutdown the server's listening socket.  And if we did, we'd just stop the server rather than trying to reach in and shutdown the conn.   Let's remove the server half of this and see if the system cooperates.